airflow/decorators/__init__.py

```diff
@@ -19,6 +19,7 @@
 
 from airflow.decorators.python import PythonDecoratorMixin, python_task  # noqa
 from airflow.decorators.python_virtualenv import PythonVirtualenvDecoratorMixin
+from airflow.decorators.sensor import sensor
```

Comment:
This module naming scheme screams gotcha to me. Probably better to either put `sensor` into the `python` module instead, or rename the module to `sensors` (plural).

Reply:
I agree with @uranusjr , personally +1 for moving it to python for consistency